pkg/generators/models/models.go

```diff
@@ -184,11 +260,12 @@ var modelTemplateSource = `// This file is auto-generated, DO NOT EDIT.
 //     Version: {{.SpecVersion}}
 package {{ .PackageName }}
 
-{{ if .Imports }}import ({{end}}
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
 {{- range .Imports}}
 	"{{.}}"
 {{ end}}
-{{- if .Imports }}){{end}}
+)
 
 {{ (printf "%s is an object. %s" .Name .Description) | commentBlock }}
```

Comment:
is is possible to extract the our templates to a separate file? it might might it easier to review and maintain

Reply:
I moved the template related portion of the code into its own .go file. Is that enough or would you like to see the templates in separate .gotmpl files?

Reply:
that works for now, maybe if we update to the next go version, we can use the new embed feature to extract them out to gotmpl
the extra complexity probably isn't needed right now